cmd/nginx-ingress/main.go

```diff
@@ -212,6 +212,11 @@ func main() {
 	}
 	glog.Infof("Starting NGINX Ingress Controller %v PlusFlag=%v", versionInfo, *nginxPlus)
 
+	unparsed := flag.Args()
+	if unparsed != nil {
+		glog.Infof("Ignoring unhandled arguments: %v", unparsed)
```

Comment:
I would use a warning
```suggestion
		glog.Warningf("Ignoring unhandled arguments: %v", unparsed)
```